### ISSUE TITLE
CANCELED is missing from valid values for update_request_status 

### ISSUE THREAD

**Issue by paulmaschhoff** [smartsheet-platform/smartsheet-python-sdk#65]
https://github.com/smartsheet-platform/smartsheet-python-sdk/blob/81882edc4889b00654b61c2a6b8b80a4e7d5a59d/smartsheet/models/sent_update_request.py#L40-L43

`CANCELED` is missing from valid values for update_request_status. This breaks the method.

``` Python
ValueError: `CANCELED` is an invalid value for SentUpdateRequest`update_request_status`, must be one of ['PENDING', 'COMPLETE']
```

**Comment by timwellswa** [comment]
fixed in PR 64

**Comment by seweil** [comment]
Fixed in release 1.2.1
